elastic/kibana
# [Infra UI] Fix process charts after refactoring

Closes #168156 
## Summary

This PR fixes process charts error: 
![image](https://github.com/elastic/kibana/assets/14139027/4d907c77-07a3-40da-96fd-2340bce2efc8)

I assume that after the merge of the refactoring PR, those files were recreated by another PR or most likely when `main` was merged to [the refactoring PR](https://github.com/elastic/kibana/pull/166965) with them as resolved conflicts. Deleting the files again fixed the issue.

## Testing
I found the issue on serverless but it probably can be reproduced on on-prem as well
On Serverless: 
- Start a local es instance: `yarn es serverless --kill --clean --license trial --ssl`
- Enable infra in the `serverless.oblt.dev.yml` file:
   - `xpack.infra.enabled: true`
- Start a local kibana instance: `yarn serverless-oblt --ssl` and see if the side nav contains the Infrastructure item
- Navigate to https://0.0.0.0:5601/ftw/app/hosts  and open Asset details flyout or page
- Click on the process tab and expand one of the processes
- The charts (CPU and Memory) should be visible
